package.json

```diff
@@ -13,49 +13,55 @@
     "dbLocation": "mongodb://localhost/didactic-disco"
   },
   "scripts": {
-    "start": "NODE_ENV=development parallelshell 'npm run express-dev-server' 'npm run webpack-dev-server'",
+    "lint": "eslint --ext .jsx app",
+    "lint:fix": "npm run lint -- --fix",
+    "pretest": "npm run lint",
+    "test": "echo 'no tests available' && exit 1",
+    "bundle": "NODE_ENV=production webpack --config webpack/config.dist.js --progress",
     "express-dev-server": "nodemon ./server/server.js",
-    "webpack-dev-server": "webpack-dev-server --colors --progress --hot --inline --config  ./webpack/config.dev.js --port 2992"
+    "webpack-dev-server": "webpack-dev-server --colors --progress --hot --inline --config  ./webpack/config.dev.js --port 2992",
+    "start": "NODE_ENV=development parallelshell 'npm run express-dev-server' 'npm run webpack-dev-server'"
   },
   "dependencies": {
-    "autoprefixer": "^6.3.6",
-    "babel-core": "^6.7.7",
-    "babel-eslint": "^6.0.4",
-    "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
     "body-parser": "^1.15.0",
     "compression": "^1.6.1",
     "connect-mongo": "^1.1.0",
-    "css-loader": "^0.23.1",
     "express": "^4.13.4",
     "express-http-proxy": "^0.6.0",
     "express-session": "^1.13.0",
-    "extract-text-webpack-plugin": "^1.0.1",
     "history": "^2.1.1",
     "lodash": "^4.11.1",
     "moment-timezone": "^0.5.4",
     "mongoose": "^4.4.14",
     "morgan": "^1.7.0",
     "node-sass": "^3.7.0",
-    "nodemon": "^1.9.2",
     "normalize.css": "^3.0.3",
     "passport": "^0.3.2",
-    "postcss-loader": "^0.9.1",
     "pug": "^2.0.0-alpha6",
     "react": "^15.0.2",
     "react-dom": "^15.0.2",
     "react-router": "^2.4.0",
-    "sass-loader": "^3.2.0",
-    "style-loader": "^0.13.1",
-    "webpack": "^1.13.0",
     "winston": "^2.2.0"
   },
   "devDependencies": {
+    "autoprefixer": "^6.3.6",
+    "babel-core": "^6.7.7",
+    "babel-eslint": "^6.0.4",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "css-loader": "^0.23.1",
     "eslint": "^2.9.0",
     "eslint-plugin-react": "^5.0.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "json-loader": "^0.5.4",
+    "nodemon": "^1.9.2",
     "parallelshell": "^2.0.0",
+    "postcss-loader": "^0.9.1",
+    "sass-loader": "^3.2.0",
+    "style-loader": "^0.13.1",
+    "webpack": "^1.13.0",
```

Comment:
I've moved everything that's not being used at run time (either client or server side) into the dev dependencies.